packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-ordering--to-es2015/decorators/exec.js

```diff
@@ -0,0 +1,31 @@
+var log = [];
```

Comment:
This test is revised from https://github.com/babel/babel/blob/89e26a0528cf557bac53c28b1a5918c2f9089681/packages/babel-plugin-proposal-decorators/test/fixtures/ordering/decorators/exec.js
It ensures that #14242 is fixed. I plan to extend this test and cover more decorator features as well as computed property keys.